[debug] [2018-07-03T01:12:47.250Z] ----------------------------------------------------------------------
[debug] [2018-07-03T01:12:47.252Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\asus\AppData\Roaming\npm\node_modules\firebase-tools\bin\firebase login:ci
[debug] [2018-07-03T01:12:47.252Z] CLI Version:   3.19.1
[debug] [2018-07-03T01:12:47.252Z] Platform:      win32
[debug] [2018-07-03T01:12:47.252Z] Node Version:  v8.11.3
[debug] [2018-07-03T01:12:47.253Z] Time:          Tue Jul 03 2018 02:12:47 GMT+0100 (W. Central Africa Standard Time)
[debug] [2018-07-03T01:12:47.253Z] ----------------------------------------------------------------------
[debug] 
[info] 
[info] Visit this URL on any device to log in:
[info] https://accounts.google.com/o/oauth2/auth?client_id=563584335869-fgrhgmd47bqnekij5i8b5pr03ho849e6.apps.googleusercontent.com&scope=email%20openid%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloudplatformprojects.readonly%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Ffirebase%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloud-platform&response_type=code&state=482501769&redirect_uri=http%3A%2F%2Flocalhost%3A9005
[info] 
[info] Waiting for authentication...
[debug] [2018-07-03T01:13:00.267Z] >>> HTTP REQUEST POST https://accounts.google.com/o/oauth2/token  
 { code: '4/AAB2L7KRJD2pqMAx7ZEEG1yQxlRxanbDOqR2dGQoMsCfZ0GHcLVdbmB6YHVFKFZOYBjJ--AVEZX7Bcj1bkXfAOM',
  client_id: '563584335869-fgrhgmd47bqnekij5i8b5pr03ho849e6.apps.googleusercontent.com',
  client_secret: 'j9iVZfS8kkCEFUPaAeJV0sAi',
  redirect_uri: 'http://localhost:9005',
  grant_type: 'authorization_code' } 
 Tue Jul 03 2018 02:13:00 GMT+0100 (W. Central Africa Standard Time)
[debug] [2018-07-03T01:13:21.280Z] Token Fetch Error: Error
    at new FirebaseError (C:\Users\asus\AppData\Roaming\npm\node_modules\firebase-tools\lib\error.js:11:16)
    at Request._callback (C:\Users\asus\AppData\Roaming\npm\node_modules\firebase-tools\lib\api.js:35:11)
    at self.callback (C:\Users\asus\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\request\request.js:185:22)
    at emitOne (events.js:116:13)
    at Request.emit (events.js:211:7)
    at Request.onRequestError (C:\Users\asus\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\request\request.js:877:8)
    at emitOne (events.js:116:13)
    at ClientRequest.emit (events.js:211:7)
    at TLSSocket.socketErrorListener (_http_client.js:387:9)
    at emitOne (events.js:116:13)
[debug] [2018-07-03T01:14:50.177Z] ----------------------------------------------------------------------
[debug] [2018-07-03T01:14:50.182Z] Command:       C:\Program Files\nodejs\node.exe C:\Users\asus\AppData\Roaming\npm\node_modules\firebase-tools\bin\firebase login --interactive
[debug] [2018-07-03T01:14:50.182Z] CLI Version:   3.19.1
[debug] [2018-07-03T01:14:50.182Z] Platform:      win32
[debug] [2018-07-03T01:14:50.182Z] Node Version:  v8.11.3
[debug] [2018-07-03T01:14:50.183Z] Time:          Tue Jul 03 2018 02:14:50 GMT+0100 (W. Central Africa Standard Time)
[debug] [2018-07-03T01:14:50.183Z] ----------------------------------------------------------------------
[debug] 
[info] 
[info] Visit this URL on any device to log in:
[info] https://accounts.google.com/o/oauth2/auth?client_id=563584335869-fgrhgmd47bqnekij5i8b5pr03ho849e6.apps.googleusercontent.com&scope=email%20openid%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloudplatformprojects.readonly%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Ffirebase%20https%3A%2F%2Fwww.googleapis.com%2Fauth%2Fcloud-platform&response_type=code&state=216594125&redirect_uri=http%3A%2F%2Flocalhost%3A9005
[info] 
[info] Waiting for authentication...
[debug] [2018-07-03T01:15:04.295Z] >>> HTTP REQUEST POST https://accounts.google.com/o/oauth2/token  
 { code: '4/AABOjqyXEzjrZ1HAn10NrD-e8MaCT-Skj-xoicBY0bmPNW8jWMePBsBzxzOONRtdF-jCiHO_44OFLyCd5pi6gw4',
  client_id: '563584335869-fgrhgmd47bqnekij5i8b5pr03ho849e6.apps.googleusercontent.com',
  client_secret: 'j9iVZfS8kkCEFUPaAeJV0sAi',
  redirect_uri: 'http://localhost:9005',
  grant_type: 'authorization_code' } 
 Tue Jul 03 2018 02:15:04 GMT+0100 (W. Central Africa Standard Time)
[debug] [2018-07-03T01:15:25.311Z] Token Fetch Error: Error
    at new FirebaseError (C:\Users\asus\AppData\Roaming\npm\node_modules\firebase-tools\lib\error.js:11:16)
    at Request._callback (C:\Users\asus\AppData\Roaming\npm\node_modules\firebase-tools\lib\api.js:35:11)
    at self.callback (C:\Users\asus\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\request\request.js:185:22)
    at emitOne (events.js:116:13)
    at Request.emit (events.js:211:7)
    at Request.onRequestError (C:\Users\asus\AppData\Roaming\npm\node_modules\firebase-tools\node_modules\request\request.js:877:8)
    at emitOne (events.js:116:13)
    at ClientRequest.emit (events.js:211:7)
    at TLSSocket.socketErrorListener (_http_client.js:387:9)
    at emitOne (events.js:116:13)
